CODE_OF_CONDUCT.md

```diff
@@ -60,8 +59,9 @@ representative at an online or offline event.
 ## Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported to the community leaders responsible for enforcement in the
-[Discord][discord] server.
+reported to the community leaders responsible for enforcement at
```

Comment:
we should not edit this document. It is the contributor covenant, much like the license, is an immutable document.

Reply:
I don't think people can DM Brain anyways unless friended, and you can't send Brain a friend request.

Reply:
This exact place says in original text:

```md
Instances of abusive, harassing, or otherwise unacceptable behavior may be
reported to the community leaders responsible for enforcement at
[INSERT CONTACT METHOD].
All complaints will be reviewed and investigated promptly and fairly.
```

In fact we already had this changed, and it was what triggered me replacing it with copy from the original source, because I noticed they're not identical. Only later on I noticed, we're actually *meant* to write our details in, so I put them back. People should be hopefully able to DM brain because that's the stated communication vector for vulnerabilities and other stuff as well. If that's not the way it is, we may want to come up with something else and place it in all incriminated places instead.